### PR TITLE
update multi-geo-capabilities-in-onedrive-and-sharepoint-online-in-of…

### PR DESCRIPTION
**Pull request by MaratMussabekov**
…fice-365.md

issue #256 
added info about SharePoint MultiGeo release

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/multi-geo-capabilities-in-onedrive-and-sharepoint-online-in-office-365.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ Users get a seamless experience when using Office 365 services, including Office
 
 Each user's OneDrive can be provisioned in or [moved by an administrator](move-onedrive-between-geo-locations.md) to a satellite location in accordance with the user's PDL. Personal files are then kept in that geo location, though they can be shared with users in other geo locations.
 
-## Sites and Groups
+## SharePoint Sites and Groups
+
+Managing of Multi-Geo feature is available through the SharePoint admin center. Detailed information could be found in the [corresponding blog post](https://techcommunity.microsoft.com/t5/Office-365-Blog/Now-available-Multi-Geo-in-SharePoint-and-Office-365-Groups/ba-p/263302).
```
Comment (JohanFreelancer9) [inline Enterprise/multi-geo-capabilities-in-onedrive-and-sharepoint-online-in-office-365.md]: **Management of the** Multi-Geo feature is available through the SharePoint admin center. Detailed information **can** be found...

Please change as indicated in **bold**.

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions he made. Thank you.

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 ,
your recommendations were applied, thank you!

**Comment by mypil** [comment]
@MikePlumleyMSFT - Can you please assist to check, approve and merge the content for this PR based on issue #256?

Thank you.

**Comment by mypil** [comment]
@MikePlumleyMSFT - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #256?

Thank you.

**Comment by robmazz** [comment]
@pamgreen-msft - I know Mike is on leave, who should be reviewing PRs for his content?

**Comment by pamgreen-msft** [comment]
Hi @JoanneHendrickson, can you review the multi-geo PRs?

**Pull request merged**